pkg/ebpf/tracer.go

```diff
@@ -112,9 +114,33 @@ func NewTracer(config *Config) (*Tracer, error) {
 		return nil, fmt.Errorf("could not start polling bpf events: %s", err)
 	}
 
+	// run a http service for expvar endpoint
+	if config.ExpVarPort > 0 {
```

Comment:
I think this belongs in `probe.go` where we set up the rest of the HTTP handlers.

Also, I think because of the way we're using the default mux, we'll end up serving /connections over HTTP as well as the unix socket if EnableDebugProfiling is enabled. 

Now that we're using HTTP for expvar, I think it's fine to use HTTP for the pprof endpoints, so fixing it should be as simple as always using a dedicated mux for the `/connections` endpoints.